### PR TITLE
Fixes #32126 - add support for additional reported data

### PR DESCRIPTION
**Pull request by ares**
The Foreman core recognizes some facts and stores them as first class
citizen, normalized data in the reported data facet since 1.24. These
are the facts missing comparing to the current set in the core. The data
can be then used cross fact sources and be displayed consistently or
used in reports.

### CODE REVIEW

File: app/services/foreman_ansible/fact_parser.rb [hunk 1]
```diff
@@ -72,6 +72,22 @@ def boot_timestamp
       Time.zone.now.to_i - facts['ansible_uptime_seconds'].to_i
     end
 
+    def virtual
+      facts['ansible_virtualization_role'] == 'guest'
+    end
+
+    def ram
+      facts['ansible_memtotal_mb'].to_i #ansible_memory_mb'].try(:[], 'real').try(:[], 'total').to_i
```
Comment (xprazak2) [inline app/services/foreman_ansible/fact_parser.rb:80]: Debugging leftover?

File: test/unit/services/fact_parser_test.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@ module ForemanAnsible
   # Host properties
   class FactParserTest < ActiveSupport::TestCase
     setup do
-      @facts_parser = ForemanAnsible::FactParser.new(facts_json)
+      @facts_parser = ForemanAnsible::FactParser.new(JSON.parse(File.read('/home/ares/z/foreman_ansible/test/fixtures/sample_facts.json')))
```
Comment (xprazak2) [inline test/unit/services/fact_parser_test.rb:10]: Debugging leftover?

**Comment by ares** [comment]
Indeed, both leftovers removed, sorry for that

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**